src/transforms/laplacian.jl

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    LaplacianSmoothing(n; λ=0.5)
+
+Perform `n` iterations of Laplacian smoothing with parameter `λ`.
+
+## References
+
+* Sorkine, O. 2005. [Laplacian Mesh Processing]
+  (http://dx.doi.org/10.2312/egst.20051044)
+"""
+struct LaplacianSmoothing{T} <: StatelessGeometricTransform
+  n::Int
+  λ::T
+
+  function LaplacianSmoothing{T}(n, λ) where {T}
+    new(n, λ)
+  end
```

Comment:
The inner constructor is only useful when we have additional lines of code to check the arguments. In this case you are only calling the built-in `new` function, so we can erase the inner constructor and it will exist by default.